Makefile

```diff
@@ -533,13 +533,8 @@ dist/argosay:
 .PHONY: kit
 kit: Makefile
 ifeq ($(shell command -v kit),)
-ifeq ($(shell uname),Darwin)
-	brew tap kitproj/kit --custom-remote https://github.com/kitproj/kit
-	brew install kit
```

Comment:
You can instead do `brew install kitproj/kit/kit` per the [Brew docs on duplicate names](https://docs.brew.sh/Taps#formula-with-duplicate-names)

Although personally I'm not particularly a fan of the `Makefile` installing system deps (such as via Brew or `pip` or `npm -g`) in the first place